storage/ops.go

```diff
@@ -1489,7 +1490,7 @@ func FindModifyInstallTargets(medias []*BlockDevice) []InstallTarget {
 // for an advanced installations.
 //	CLR_BOOT = The /boot partition; must be vfat
 //	CLR_SWAP = A swap partition to use; can be more than one
-//	CLR_ROOT = The / root partition; must be ext[234] or xfs
+//	CLR_ROOT = The / root partition; must be EXT[234], XFS or F2FS.
```

Comment:
Just was curious about this one, is there a specific reason you wanted them lower case? To my understanding abbreviations should always be capitalized, and especially when it will be shown as apart of the GUI

Reply:
EXT isn't an abbreviation though, it is a file system type; ext2, ext3, ext4.
Technically, yes XFS may be more correct than xfs, but also all of the commands, libraries, and man pages in Linux are all in lower-case -- hence my preference.